### ISSUE TITLE
jsdoc 3.3.2 is deprecated

### ISSUE THREAD

**Issue by bmsdave** [allenhwkim/angular-jsdoc#100]
Hi! I find bug in dependensies:

```
$: npm install angular-jsdoc
npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm ERR! fetch failed https://github.com/ariya/esprima/tarball/49a2eccb243f29bd653b11e9419241a9d726af7c
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://github.com/ariya/esprima/tarball/49a2eccb243f29bd653b11e9419241a9d726af7c
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://github.com/ariya/esprima/tarball/49a2eccb243f29bd653b11e9419241a9d726af7c
```

**Comment by bmsdave** [comment]
"npm install angular-jsdoc@1.4.1" - is fine.
but "npm install angular-jsdoc@1.4.2" - is bad

**Comment by schmod** [comment]
Closed by 883a98a.  

Will be fixed in next release.
